atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/AbstractDbWriteTable.java

```diff
@@ -119,6 +123,41 @@ public void putSentinels(Iterable<Cell> cells) {
     }
 
     @Override
+    public void update(Cell cell, byte[] oldValue, byte[] newValue) {
+        String prefixedTableName = prefixedTableNames.get(tableRef);
+        long ts = AtlasDbConstants.TRANSACTION_TS;
```

Comment:
wondering if this should be passed in to update, since update appears to be more generic then checkAndSet